ftl/core/importing.ftl

```diff
@@ -116,6 +116,54 @@ importing-cards-added =
        *[other] { $count } cards added.
     }
 importing-file-empty = The file you selected is empty.
+importing-notes-added =
+    { $count ->
+        [one] { $count } new note was successfully imported into your collection.
+       *[other] { $count } new notes were successfully imported into your collection.
+    }
+importing-notes-updated =
+    { $count ->
+        [one] { $count } was updated to the latest version.
+       *[other] { $count } were updated to the latest version.
+    }
+importing-existing-notes-skipped =
+    { $count ->
+        [one] { $count } was not imported because it already exists identically in your collection.
+       *[other] { $count } were not imported because they already exist identically in your collection.
+    }
+importing-conflicting-notes-skipped =
+    { $count ->
+        [one] { $count } was not imported because its note type has changed.
+       *[other] { $count } were not imported because their note type has changed.
+    }
+importing-import-log = Import Log
+importing-import-log-for-file = Import Log for { $val }
+importing-no-notes-in-file = No notes found in file.
+importing-num-notes-found-in-file =
+    { $count ->
+        [one] { $count } note was found in the file.
+       *[other] { $count } note were found in the file.
+    }
+importing-notes-part-of-collection =
+    {
+        $count ->
+            [one] { $count } ({ $percent }%) is now part of your collection.
+           *[other] { $count } ({ $percent }%) are now part of your collection.
+    }
```

Comment:
Hi Damien, the bold first sentence was my idea. I wanted to present the most important information first to the user. In my opinion this was the "success rate": How many of the notes in this file are part of my collection and can be learned in the future? (= sum of: a) newly added + b) updated + c) already in collection) But I see how this could be confusing to other users. I'm probably trying too hard to prevent users from interpreting the sentence `x notes were not imported because they are already part of your collection` as a failure (and then try to import the same file again with the same result). But that's simply what I encountered a lot with new users.

I agree that "are now" somehow suggests a change and is therefore suboptimal. Your suggested non bold first sentence with "Of these:" seems fine to me. To aid understanding I also think it would be nice to slightly indent the detailed summary sentences below the first sentence like you did above.

Adding the amount of cards just in the first line seems like a very good idea to me but I would do it in brackets like this: "`23 notes (= 46 cards) found in the file.`" in order to highlight that cards are not something that is added separately, besides the notes.

I would also link the words "notes" and "cards" to corresponding sections of the manual so people can look up the difference.

Reply:
> I'm probably trying too hard to prevent users from interpreting the sentence x notes were not imported because they are already part of your collection as a failure 

I understand the motivation; it has been a source of confusion in the past. That said, I'm hoping that the ability to easily view the cards in the browse screen will largely reduce that confusion.

> 23 notes (= 46 cards) found in the file.

That's better, but how about we drop the '=', and just use the brackets?

Reply:
>That said, I'm hoping that the ability to easily view the cards in the browse screen will largely reduce that confusion.
That's a good point.

>how about we drop the '=', and just use the brackets?
I'd keep them for extra clarity but am fine with whatever you choose. 👌

Reply:
I might be nitpicking here, but I'd prefer without the '=', as that implies a stronger correspondence between the two, and it's not always the case that 1 note = 2 cards.